.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -449,40 +439,31 @@ jobs:
       uses: seanmiddleditch/gha-setup-vsdevenv@v4
 
     # Ninja is required on Windows for CMAKE_CXX_COMPILER_LAUNCHER to work for using sccache.
-    #
-    # Until debug and release packages are split we only build release packages for vcpkg
-    # to improve CI run times, hence VCPKG_HOST_TRIPLET and VCPKG_TARGET_TRIPLET
-    # https://github.com/microsoft/vcpkg/issues/1626
     - name: "Configure"
       run: >-
         cmake
         -D USE_QT5=${{ matrix.qt_version == 5 }}
         -D CMAKE_BUILD_TYPE=Release
-        -D VCPKG=${{ matrix.vcpkg }}
-        -D VCPKG_HOST_TRIPLET=${{ matrix.vcpkg_triplet }}
-        -D VCPKG_BUILDTREES_ROOT=${{ matrix.vcpkg_buildtrees_root  }}
-        -D VCPKG_TARGET_TRIPLET=${{ matrix.vcpkg_triplet }}
         -G Ninja
         -S . -B build
-      working-directory: ${{ matrix.workspace }}
       env:
         RUSTC_WRAPPER: sccache
         CC: ${{ matrix.cc }}
         CXX: ${{ matrix.cxx }}
     - name: "Build"
       run: cmake --build build --config Release --parallel ${{ matrix.cores }}
-      working-directory: ${{ matrix.workspace }}
       env:
         RUSTC_WRAPPER: sccache
     - name: "Test"
       run: ctest ${{ matrix.ctest_args }} -C Release -T test --output-on-failure --parallel ${{ matrix.cores }}
-      working-directory: ${{ matrix.workspace }}/build
+      working-directory: ./build
       env:
         # Use the version of clang-format from pip
         CLANG_FORMAT_BINARY: ${{ matrix.clang_format_path }}
         RUSTC_WRAPPER: sccache
         QT_QPA_PLATFORM: ${{ matrix.qt_qpa_platform }}
         QT_SELECT: qt${{ matrix.qt_version }}
+        DYLD_FRAMEWORK_PATH: ${{ matrix.dyld_framework_path }}
```

Comment:
This is needed so it can find the dynamic libraries from Qt on MacOS, right?
Is this still needed after the fixes by @ahayzen-kdab ?

Reply:
Right, this is needed for `cargo test` to work in certain places where there is linking going on to dyn libs, but they are not going through CxxQtBuilder (which sets up the rpath). Think of this as effectively LD_LIBRARY_PATH.

Reply:
Ah, so it's needed for the tests that use qt types, but use CXX builder, not CxxQtBuilder?
That makes sense.